easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -352,7 +352,7 @@ private static Activity getActivity(@NonNull Object object) {
         } else if (object instanceof android.app.Fragment) {
             return ((android.app.Fragment) object).getActivity();
         } else {
-            return null;
+            throw new IllegalArgumentException("Context cannot be null.");
```

Comment:
Wouldn't this message be more accurate as "Object must be an Activity or a Fragment"?  This `else` does not imply nullity.